ios/Egendata/Info.plist

```diff
@@ -34,7 +34,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>45</string>
+	<string>56</string>
```

Comment:
was this autogenerated by test runs? 

Do we need to set this to 0 when we merge or this is correct version matching what is in Testflight?

Reply:
So let's try it live? 😄